src/utils/tests.jsx

```diff
@@ -25,28 +24,8 @@ export function renderWithRouter(
   };
 }
 
-/* eslint-disable react/prop-types */
-export const FakeAppContext = ({
-  initialAppState = {},
-  children,
-}) => (
-  <AppContext.Provider value={initialAppState}>
+export const renderWithSearchContext = (children) => renderWithRouter(
```

Comment:
Can you do me a favor and export this (from the package)? I want to use it in other repos. ;)